src/renderer/actions.ts

```diff
@@ -1285,6 +1286,7 @@ export function openWorkspace(workspacePath?: string | null): ThunkAction {
             } else {
                 dispatch(setSelectedWorkspaceResourceName(null));
             }
+            dispatch(savePreferences());
```

Comment:
Why? Preferences are actually saved automatically. Is this to avoid losing this setting after a crash before preferences are saved next time?

Reply:
I found that when I open a workspace, log out and then log in again with the reopenlastworkspace option set, the workspace is not loaded in again.